lib/test/test_hamburger.rb

```diff
@@ -15,7 +15,6 @@ def test_level_teacher_en
     assert_equal visibility[:hamburger_class],          Hamburger::SHOW_ALWAYS
     assert_equal visibility[:show_teacher_options],     Hamburger::SHOW_ALWAYS
     assert_equal visibility[:show_student_options],     Hamburger::HIDE_ALWAYS
-    assert_equal visibility[:show_signed_out_options],  Hamburger::HIDE_ALWAYS
```

Comment:
Removed these since they are no longer relevant now that the `signed_out_entries` list was removed.